### PR TITLE
Create an integrations overview page for MS tools

### PR DESCRIPTION
**Pull request by jasonblais**
To support an upcoming blog post

### CODE REVIEW

File: source/deployment/microsoft-integrations.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect Microsoft tools in Mattermost. Here are some popular options:
+
+Office 365 Calendar (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Two-way integration between Mattermost and Office 365 Calendar, developed by Mattermost.
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Reflect user status as "Do Not Disturb' when in a meeting scheduled via Outlook Calendar.
+- Supports <XXX?>
```
Comment (justinegeffen) [inline source/deployment/microsoft-integrations.rst:12]: Should we remove this or is it a placeholder?

File: source/deployment/microsoft-integrations.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- Source code + docs: https://github.com/mattermost/mattermost-plugin-mscalendar
```
Comment (justinegeffen) [inline source/deployment/microsoft-integrations.rst:13]:
```suggestion
- Source code + docs: https://github.com/mattermost/mattermost-plugin-mscalendar.
```

File: source/deployment/microsoft-integrations.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- Start and join voice calls, video calls and use screensharing with your team members, developed by kosgrz and maintained by Mattermost.
+- Clicking a video icon in a Mattermost channel invites team members to join a Skype for Business call, hosted using the credentials of the user who initiated the call.
+- Supports a vendor-hosted cloud solution.
+- Source code + docs: https://github.com/kosgrz/mattermost-plugin-skype4business.
+
+Office 365 SSO (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Configure Mattermost to use your Office 365 logon credentials and Azure Active Directory account as a single sign-on (SSO) service, developed by Mattermost.
+- Supports Microsoft Active Directory Tenants for team creation, account creation and sign-in.
```
Comment (justinegeffen) [inline source/deployment/microsoft-integrations.rst:27]:
```suggestion
- Supports Microsoft Active Directory Tenants for team creation, account creation, and log in.
```
Not sure this is linked to SSO or refers to signing in.

File: source/deployment/microsoft-integrations.rst [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Docs: https://docs.mattermost.com/deployment/sso-office.html
```
Comment (justinegeffen) [inline source/deployment/microsoft-integrations.rst:29]:
```suggestion
- Docs: https://docs.mattermost.com/deployment/sso-office.html.
```

File: source/deployment/microsoft-integrations.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Collabora Online
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- View Microsoft Doc (.doc and .docx), Excel (.xls and .xlsx) and PowerPoint (.ppt and .pptx) files in Mattermost, developed by Collabora.
```
Comment (justinegeffen) [inline source/deployment/microsoft-integrations.rst:34]:
```suggestion
- View Microsoft Word (.doc and .docx), Excel (.xls and .xlsx), and PowerPoint (.ppt and .pptx) files in Mattermost, developed by Collabora.
```

File: source/deployment/microsoft-integrations.rst [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- Allows multiple people to edit an Office 365 file at the same time from Mattermost, replacing the original file on the Mattermost server.
+- Self-hosted solution for file sharing and management of Office 365 files.
+- Source code + docs: https://github.com/CollaboraOnline/collabora-mattermost
```
Comment (justinegeffen) [inline source/deployment/microsoft-integrations.rst:37]:
```suggestion
- Source code + docs: https://github.com/CollaboraOnline/collabora-mattermost.
```

File: source/deployment/microsoft-integrations.rst (hunk 1 above)
Comment (jasonblais) [reply source/deployment/microsoft-integrations.rst:12]: Good catch, meant as a question/placeholder for @aaronrothschild

Comment (aaronrothschild) [reply source/deployment/microsoft-integrations.rst:12]: LGTM

File: source/deployment/microsoft-integrations.rst [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+- Supports <XXX?>
+- Source code + docs: https://github.com/mattermost/mattermost-plugin-mscalendar.
+
+Skype for Business
+~~~~~~~~~~~~~~~~~~~~
+
+- Start and join voice calls, video calls, and use screen-sharing with your team members. Developed by kosgrz and maintained by Mattermost.
+- Clicking a video icon in a Mattermost channel invites team members to join a Skype for Business call, hosted using the credentials of the user who initiated the call.
+- Supports a vendor-hosted cloud solution.
+- Source code + docs: https://github.com/kosgrz/mattermost-plugin-skype4business.
```
Comment (aaronrothschild) [inline source/deployment/microsoft-integrations.rst:21]:
```suggestion
- Source code + docs: https://github.com/mattermost/mattermost-plugin-skype4business
```

**Comment by amyblais** [comment]
Will this be added to TOC later?
Also "do not disturb" is surrounded by different quotes.

Screenshot at Jun 08 08-22-28

**Comment by jasonblais** [comment]
@amyblais good catch..fixing both at https://github.com/mattermost/docs/pull/3673

**Pull request merged**